server/src/models/services/populate.js

```diff
@@ -2,56 +2,100 @@ import _ from "lodash";
 
 import { get_standard_csv_file_rows } from "../load_utils.js";
 
-const multi_value_string_fields_to_arrays = (list_fields, seperator = ",") =>
+const multi_value_string_fields_to_arrays = (list_fields, seperator = "<>") =>
   _.mapValues(list_fields, (array_string) => _.split(array_string, seperator));
-
+const convert_to_bool_or_null = (value, true_val, false_val) => {
+  if (value === true_val) {
+    return true;
+  } else if (value === false_val) {
+    return false;
+  } else {
+    return null;
+  }
+};
 export default async function ({ models }) {
-  const { ServiceStandard, Service } = models;
+  const { ServiceStandard, Service, ServiceReport, StandardReport } = models;
+
+  const service_report_rows = _.map(
+    get_standard_csv_file_rows("service-report.csv"),
+    ({
+      id: standard_id,
+      cra_business_ids_collected,
+      sin_collected,
+      comment: service_report_comment,
+
+      ...other_fields
+    }) => ({
+      standard_id,
+      cra_business_ids_collected: convert_to_bool_or_null(
+        cra_business_ids_collected,
+        "Yes",
+        "No"
+      ),
+      sin_collected: convert_to_bool_or_null(sin_collected, "Yes", "No"),
+      service_report_comment,
+      ...other_fields,
+    })
+  );
+  const standard_report_rows = _.map(
+    get_standard_csv_file_rows("standard-report.csv"),
+    ({ is_target_met, comment: standard_report_comment, ...other_fields }) => ({
+      is_target_met: convert_to_bool_or_null(_.toInteger(is_target_met), 1, 0),
```

Comment:
not sure if we should just ask Ross to put 1/0 or true/false for these. Also specifically this one might make sense to put as `met` or `not_met` since we use those keys to index icons (`services/shared.js`)